src/main/resources/docs/csrf-protection.md

```diff
@@ -47,6 +47,26 @@ and pass it back as either `X-CSRF-TOKEN` or `X-XSRF-TOKEN`. The only difference
 the value of `X-CSRF-TOKEN` is considered to be unencrypted while the value of `X-XSRF-TOKEN` is considered
 to be encrypted and hence it will be decrypted before validating the accuracy of the token.
 
+In the following example, we pass `X-CSRF-TOKEN` in the header of an Ajax call. 
+
+```javascript
```

Comment:
We should mention that this only works inside a Pebble template and not outside (like in a Vue component of a .js file)

Reply:
ok - I added some pebble verbiage. Do you think it needs an info section saying more explicitly?

Reply:
I think it will be fine without